goci-interfaces/goci-ui/src/main/resources/logback-dev.xml

```diff
@@ -25,6 +25,5 @@
 
     <root level="INFO">
         <appender-ref ref="CONSOLE" />
-        <appender-ref ref="FILE" />
```

Comment:
I think we should keep both console & file, sometimes exception during startup need to captured in log files